### PR TITLE
Use wagtail-markdown instead of custom code

### LINKED ISSUE #44: Rework markdown handling
While developing the proof of concept, @T4rikA and me tried to use https://github.com/torchbox/wagtail-markdown for markdown rendering. There were some problems where the library was not flexible enough for our approach. We decided to patch these issues inside this project to deliver a PoC quickly. These issues should be identified and ideally patched in the upstream repository.

### PR DESCRIPTION
**Pull request by jeriox**
closes #44 

ToDo

- [x] fix ToC and abbr rendering
- [x] add translation stuff to MarkdownField (https://github.com/torchbox/wagtail-markdown/issues/102)
- [ ] add buttons to editor (https://github.com/torchbox/wagtail-markdown/issues/64)

### CODE REVIEW

**Comment by jeriox** [comment]
so in theory this would be ready, but for whatever reason the CI is flaky AF on this PR. It keeps complaining that setuptools is not available in the environment although it is. In one place, upgrading setuptools and the dependency in question helped, but not in the next place

File: tests/test_minutes_extensions.py [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-            result,
-            [
-                "*12:00: First Last enters the meeting*  ",
-                "*12:00: Prof. First Last enters the meeting*  ",
-                "*12:00: Prof. First Last enters the meeting via Means*  ",
-                "*12:00: Prof. First Last leaves the meeting*  ",
-                "*12:00: Prof. First Last (Special Role) enters the meeting*  ",
-            ],
-        )
+# import django
```
Comment (frcroth) [inline tests/test_minutes_extensions.py:1]: Any way we can still use this file?

**Review by frcroth — COMMENTED** [review]
Needs to be rebased, since the migrations changed

File: tests/test_minutes_extensions.py (hunk 1 above)
Comment (jeriox) [reply tests/test_minutes_extensions.py:1]: i wanted to wait for your PR with tests, as it failed here. will merge and try to get it working

**Comment by jeriox** [comment]
@frcroth any idea why the installation of bootstrap suddenly fails with
```
Traceback (most recent call last):
  File "install-bootstrap.py", line 7, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

**Comment by frcroth** [comment]
> @frcroth any idea why the installation of bootstrap suddenly fails with
> 
> ```
> Traceback (most recent call last):
>   File "install-bootstrap.py", line 7, in <module>
>     import requests
> ModuleNotFoundError: No module named 'requests'
> ```

Because you moved `poetry install` behind the bootstrap installation

**Comment by jeriox** [comment]
> > @frcroth any idea why the installation of bootstrap suddenly fails with
> > ```
> > Traceback (most recent call last):
> >   File "install-bootstrap.py", line 7, in <module>
> >     import requests
> > ModuleNotFoundError: No module named 'requests'
> > ```
> 
> Because you moved `poetry install` behind the bootstrap installation

thx! wasn't aware that it depends on stuff that we install

**Comment by jeriox** [comment]
@frcroth as your PR at wagtail-markdown ist still not reviewed, I would vote for merging this PR and therefore closing #44 while keeping #117 open and build a PR that reintroduces the buttons once the dependency is ready

**Review by frcroth — APPROVED** [review]

**Pull request merged**